exporters/aerospike/e2e/e2e_spec.yml

```diff
@@ -0,0 +1,46 @@
+description: |
+  End-to-end tests for Aerospike integration
+
+scenarios:
+  - description: |
+      This scenario will verify that metrics from aerospike prometheus exporter are correcly collected.
+    before:
+      - docker-compose up -d
+    after:
+      - docker-compose down -v
+    integrations:
+      - name: nri-aerospike
+        binary_path: ../target/bin/linux_amd64/nri-aerospike
+        exporter_binary_path: ../target/bin/linux_amd64/aerospike-exporter
+        config:
+          aerospike_cluster_name: aerospike_cluster_name
+          aerospike_db_host: localhost
+          aerospike_db_port: 3000
+          exporter_port: 9145
+          label_type: development
+          label_source: aerospike
+          exporter_files_path: /tmp
+
+    tests:
+      nrqls:
+        - query: "FROM Metric SELECT latest(aerospike_node_stats_client_connections)"
+      entities:
+        - type: "AEROSPIKE_NODE"
+          data_type: "Metric"
+          metric_name: "aerospike_node_stats_client_connections"
+        - type: "AEROSPIKE_NAMESPACE"
+          data_type: "Metric"
+          metric_name: "aerospike_namespace_available_bin_names"
+      metrics:
+        - source: "aerospike.yml"
+          except_entities:
+            - AEROSPIKE_NAMESPACE
+          except_metrics:
+            - aerospike_namespace_available_bin_names
+            - aerospike_namespace_allow_ttl_without_nsup
+            - aerospike_namespace_appeals_records_exonerated
+            - aerospike_namespace_appeals_rx_active
+            - aerospike_namespace_appeals_tx_active
+            - aerospike_namespace_appeals_tx_remaining
+            - aerospike_namespace_available_bin_names
+            - aerospike_namespace_cache_read_pct
```

Comment:
All these metrics seem to be created by the test environment so they shouldn't be excluded